### PR TITLE
refactor: add root event handler more readable

### PR DESCRIPTION
**Pull request by yuanyan**

### CODE REVIEW

File: packages/lexical/src/LexicalEvents.ts [hunk 1]
```diff
@@ -118,35 +118,31 @@ import {
...
-  ['cut', PASS_THROUGH_COMMAND],
-  ['copy', PASS_THROUGH_COMMAND],
-  ['dragstart', PASS_THROUGH_COMMAND],
-  ['dragover', PASS_THROUGH_COMMAND],
-  ['dragend', PASS_THROUGH_COMMAND],
-  ['paste', PASS_THROUGH_COMMAND],
-  ['focus', PASS_THROUGH_COMMAND],
-  ['blur', PASS_THROUGH_COMMAND],
-  ['drop', PASS_THROUGH_COMMAND],
+  ['keydown', onKeyDown as RootElementEventHandler],
```
Comment (acywatson) [inline packages/lexical/src/LexicalEvents.ts:126]: It seems weird that we have to cast this?

**Review by acywatson — COMMENTED** [review]
This seems fine to me, aside from the casting. Not sure why we had that PASS_THROUGH_COMMAND. Let's see if others find it an improvement, as well.

**Comment by trueadm** [comment]
I'm not really sure this makes the handling logic more readable as there seems more indirection and more casting going on.

**Comment by acywatson** [comment]
> I'm not really sure this makes the handling logic more readable as there seems more indirection and more casting going on.

PASS_THROUGH_COMMAND seems more indirect to me. Why should there be frozen empty objects in the list of events instead of just event handlers?

That said, I'm not bothered either way tbh

**Comment by acywatson** [comment]
Thanks for the contribution here, but I don't really see the benefits outweighing the potential costs here. I agree that the current PASS_THROUGH situation makes very little sense from a readability standpoint, but the risk of a performance regression here outweighs the benefits of rearranging this code, I think.

**Pull request closed without merging**